similar/main/titles.cpp

```diff
@@ -1529,6 +1536,24 @@ static window_event_result briefing_handler(window *, const d_event &event, brie
 			}
 			break;
 
+		case EVENT_JOYSTICK_BUTTON_DOWN:
+			// using joy_translate_menu_key doesn't work here for unclear
```

Comment:
How does it fail?

Reply:
The game hangs with a black screen on level entry (before the "Prepare for Descent" box appears, but after the briefing screens). The stack trace shows multiple levels of calls within the event system; to be honest, I just didn't investigate further at this point and came up with that alternative approach instead.

Reply:
That stack trace is somewhat reasonable if the game is waiting for something to make forward progress.  Perhaps it got confused and was not able to make forward progress.